src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,103 @@
+@import "/styles/utils/reset.scss";
+@import "/styles/utils/variables.scss";
+@import "/styles/blocks/stars.scss";
+
+.container {
+  width: $card-width;
+  margin-right: 50px;
```

Comment:
do not set external geometry to bem blocks

Reply:
In this case container is a wrap to add external sizes on it, but not to product-card (which is bam block)

Reply:
it is better to use bem mixes instead of add extra tags